vendor.yml

```diff
@@ -55,7 +58,10 @@ css:
   - node_modules/rdash-ui/dist/css/rdash.min.css
   - node_modules/isteven-angular-multiselect/isteven-multi-select.css
   - node_modules/ui-select/dist/select.min.css
-  - node_modules/font-awesome/css/font-awesome.min.css
+  - node_modules/@fortawesome/fontawesome-free-webfonts/css/fa-brands.css
```

Comment:
Do we actually need all of these? Or only some? Because I believe we're not using any brand icon actually (might be wrong though). Do we need solid/regular as well?

Reply:
Yes, we need all of these, because of some icon from the previous version using prefix "fa" need to change to "far" instead of "fas". And for the brand icon, yes we use some of the icons like an example in the about page.